raaghu-elements/src/rds-select-list/rds-select-list.tsx

```
@@ -279,6 +276,53 @@ const RdsSelectList = (props: RdsSelectProps) => {
+          colorVariant="primary"
+          width="16px"
+          height="16px"
+          borderWidth="2px"
+          size={SpinnerSize.Small}
+        />
+      </div>
+    ),
+    IndicatorSeparator: () => null
+  };
+
+  const handleInputChange = (inputValue: string) => {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)